content/blog/guides/2020/2020-04-28-continous-integration-with-netlify-build-plugins/index.mdx

```diff
@@ -0,0 +1,82 @@
+---
+title: Continuous Integration with Netlify Build Plugins
+date: '2020-04-28T19:15:38.655Z'
+slug: '/continuous-integration-with-netlify-build-plugins'
+desription: 'With the new Netlify build plugins it just got super simple to create a CI pipeline'
+draft: true
+---
+
+## What are you talking about?
+
+Okay, so let me get this straight: This blog is deployed on Netlify because we love Netlify ❤️. Seriously, they put just a nice focus on everything you need to be a happy developer: It’s well integrated with GitHub, you get fancy preview URLs for every single pull request, you can run your website with zero configuration and beyond that they offer a great Netlify CLI and API in case you want to hack their system. So everything is great? Almost! The thing is, I don’t work on this blog alone and so we needed a continuous integration (CI) pipeline to keep our blog running smoothly. We do boring stuff like running unit tests, linting the code or check the `yarn.lock` file for uncommitted changes. As I said, boring stuff but really important. In order to get this pipeline we configured a GitHub workflow to run all the checks for every pull request. We, of course, also run the workflow with every merge on master so we ended up disabling Netlify’s great **auto publishing** feature and used a GitHub action to deploy the blog manually within the workflow. Sure it works, but it feels messy and I was super unhappy to loose the full power of the Netlify ecosystem.
```

Comment:
+ Nice introduction but I would keep it a bit shorter.. I would be more precise about what the post is about here.
+ I would also explicitly write that we removed GithubActions it's not really clear to me in the text
+ `loose` === `lose`